docs/book/features/volume_health.md

```diff
@@ -0,0 +1,112 @@
+Volume Health will pull the volume healths status of Persistent Volume from the underlying vSphere storage to update the volume health annotation on the Persistent Volume Claim at a default interval of 5 minutes. This interval can be configured by defining environment variable `VOLUME_HEALTH_INTERVAL_MINUTES` in the deployment YAML file for `vsphere-syncer` container. The following example configures the interval to 10 minutes.
+
+```bash
+...
+ - name: vsphere-syncer
+          ...
+          env:
+            - name: CLUSTER_FLAVOR
+              value: "WORKLOAD"
+            - name: KUBERNETES_SERVICE_HOST
+              value: "127.0.0.1"
+            - name: KUBERNETES_SERVICE_PORT
+              value: "6443"
+            - name: FULL_SYNC_INTERVAL_MINUTES
+              value: "30"
+            - name: VOLUME_HEALTH_INTERVAL_MINUTES
+              value: "10"
+...
+```
+
+
+## Volume Health Status Example
+
+Create a PVC in the Tanzu Kubernetes Grid Cluster.
+
+```bash
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: example-vanilla-block-pvc
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Mi
+  storageClassName: gc-storage-profile
+```
+
+```bash
+kubectl apply -f example-pvc.yaml
+```
+
+After the PVC in the Tanzu Kubernetes Grid Cluster is bounded to the PV, check the corresponding PVC in the Project Pacific Cluster to make sure that PVC is also bounded to a PV.
+
+```bash
+kubectl get pvc --all-namespaces
+NAMESPACE             NAME                                                                        STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS         AGE
+test-gc-e2e-demo-ns   8dee9083-cf01-4ab0-a1a7-b6fce5e17133-07367416-d8b9-4a6d-922e-79362ef08277   Bound    pvc-c6f6e1a9-66e6-405c-b9d5-f2716c5630b6   1Mi        RWO            gc-storage-profile   4m53s
+```
+Check the volume health annotation on the above PVC, it shows `accessible`.
+
+```bash
+kubectl describe pvc 8dee9083-cf01-4ab0-a1a7-b6fce5e17133-07367416-d8b9-4a6d-922e-79362ef08277 -n test-gc-e2e-demo-ns
+Name:          8dee9083-cf01-4ab0-a1a7-b6fce5e17133-07367416-d8b9-4a6d-922e-79362ef08277
+Namespace:     test-gc-e2e-demo-ns
+StorageClass:  gc-storage-profile
+Status:        Bound
+Volume:        pvc-c6f6e1a9-66e6-405c-b9d5-f2716c5630b6
+Labels:        <none>
+Annotations:   pv.kubernetes.io/bind-completed: yes
+               pv.kubernetes.io/bound-by-controller: yes
+               volume.beta.kubernetes.io/storage-provisioner: csi.vsphere.vmware.com
+               volumehealth.storage.kubernetes.io/messages: accessible
+Finalizers:    [kubernetes.io/pvc-protection]
+Capacity:      1Mi
+Access Modes:  RWO
+VolumeMode:    Filesystem
+Mounted By:    <none>
+...
+```
+
+Then check the volume health annotation on the PVC in Tanzu Kubernetes Grid Cluster, it also shows `accessible`.
+
+```bash
+kubectl describe pvc example-vanilla-block-pvc
```

Comment:
Since volume health is not supported on vanilla cluster, we should remove "vanilla" from the example.  There are a few other places in this doc that uses "vanilla" as well that should be removed.